src/common/utils/getCourseKeyString/index.ts

```diff
@@ -0,0 +1,5 @@
+import { CourseKey } from '@/common/utils/types'
+
+export function getCourseKeyString(course: CourseKey): string {
+  return `${course.courseNo}-${course.studyProgram}-${course.academicYear}-${course.semester}`
```

Comment:
Will this face same problem as #260 ?

Reply:
No. There will be only one course card per course.